src/components/Course/Index.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="index">
+    <el-row style="background-color: #A40006">
+      <el-col :span="2" style="float: right"><v-nav></v-nav></el-col>
+    </el-row>
+    <el-row style="height: 100%" :gutter="20">
+      <el-col :span="4" style="margin-right: 10%">
+          <el-menu class="el-menu-vertical-demo" @open="handleOpen" @close="handleClose" :collapse="isCollapse" style="float: left;margin: 0 5% 0 0">
```

Comment:
you had better make the whole el-menu to be a single component(el-menu is the father tag)  but not put the v-aside in el-menu
so you should write in this way
`
this is your aside vue file
<template>
<el-menu>
   // the old aside code
<el-menu>
</template>
`
to avoid some strange problems

Reply:
I will do it when  the collapse function is add to aside